tilequeue/query.py

```diff
@@ -64,21 +63,26 @@ def jinja_filter_bbox(bounds, srid=900913):
     return bbox
 
 
-def build_feature_queries(bounds, layer_data, zoom):
+def build_feature_queries(unpadded_bounds, layer_data, zoom):
     queries_to_execute = []
     for layer_datum in layer_data:
+        query_bounds_pad_fn = layer_datum['query_bounds_pad_fn']
+        if query_bounds_pad_fn:
+            padded_bounds = query_bounds_pad_fn(unpadded_bounds)
```

Comment:
Are the `unpadded_bounds` are in mercator coords? Should we be applying some sort of conversion factor to in the padding process, since the config file has them in 1/256ths of a tile?